packages/@lwc/engine/src/framework/membrane.ts

```diff
@@ -15,6 +17,7 @@ export const reactiveMembrane = new ObservableMembrane({
     valueObserved,
     valueMutated,
     valueDistortion,
+    membraneTag,
```

Comment:
this option is not yet available in observable membrane, but it is coming. PR https://github.com/salesforce/observable-membrane/pull/48